bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/i18n/sonnen_fr.properties

```diff
@@ -1,7 +1,7 @@
-# add-on
+# binding
 
-addon.sonnen.name = Extension Sonnen
-addon.sonnen.description = Cette extension communique avec une batterie solaire Sonnen.
```

Comment:
Please revert these changes as well.

Reply:
I cannot revert them. The latest version I have in the repo is the one I reverted it to. According to git it is the commit from the openhab-bot. If you want me to further revert it, I can just delete it.

Reply:
This is the current version in Git:
https://github.com/openhab/openhab-addons/blob/b3d3d20b8bb6d486940efc17c3c381982e7e71cb/bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/i18n/sonnen_fr.properties#L1-L4

Perhaps you need to rebase towards main after pulling latest changes?

Reply:
solved